### PR TITLE
Layers for patterns

### PR DESCRIPTION
**Pull request by pflopez**
- Adds patterns for sequenced layers
- Updates the apply treatment modal to show different use case when editing treatments
- Pattern images are placeholders

### CODE REVIEW

File: src/interface/src/app/treatments/treatment-layout/treatment-layout.component.ts [hunk 1]
```diff
@@ -73,58 +76,6 @@ export class TreatmentLayoutComponent {
...
-    this.router.events
-      .pipe(
-        untilDestroyed(this),
-        filter((event) => event instanceof NavigationEnd) // Only react to navigation events
-      )
-      .subscribe(() => {
-        const data = getMergedRouteData(this.route.snapshot);
-        const projectAreaId = data['projectAreaId'];
-
-        if (data) {
```
Comment (pflopez) [inline src/interface/src/app/treatments/treatment-layout/treatment-layout.component.ts:91]: this stuff was a bit confusing, moved most stuff to loadTreatmentByRouteData and refactored treatmentsState a bit to hopefully clarify!

File: src/interface/src/app/treatments/treatments.state.ts [hunk 2]
```diff
@@ -61,65 +63,83 @@ export class TreatmentsState {
...
-  }) {
-    this._scenarioId = opts.scenarioId;
-    this._treatmentPlanId = opts.treatmentId;
-    this._projectAreaId$.next(opts.projectAreaId);
+  /**
+   * Uses route data to load summary and treatment plan
+   * returns true when both are loaded.
+   * @param data
+   */
+  loadTreatmentByRouteData(data: TreatmentRoutingData) {
```
Comment (pflopez) [inline src/interface/src/app/treatments/treatments.state.ts:71]: might be easier to diff this side-by-side, git is scrambling all the methods by line number

File: src/interface/src/app/treatments/treatment-map/treatment-map.component.ts [hunk 3]
```diff
@@ -152,6 +154,14 @@ export class TreatmentMapComponent {
     this.mouseStart = event;
   }
 
+  mapLoaded(event: MapLibreMap) {
+    this.mapLibreMap = event;
+  }
+
+  onImageMissing(a: any) {
+    // console.log('missing!', a);
```
Comment (lastminutediorama) [inline src/interface/src/app/treatments/treatment-map/treatment-map.component.ts:162]: Did you want to do more with this? Or is this just a placeholder for the future?

File: src/interface/src/app/treatments/map-stands/map-stands.component.ts [hunk 4]
```diff
@@ -72,16 +81,22 @@ export class MapStandsComponent implements OnChanges, OnInit {
 
   selectedStands$ = this.selectedStandsState.selectedStands$;
   treatedStands$ = this.treatedStandsState.treatedStands$;
+  sequencedStandsId$ = this.treatedStandsState.sequenceStandsIds$;
```
Comment (lastminutediorama) [inline src/interface/src/app/treatments/map-stands/map-stands.component.ts:84]: did you mean to call this sequencedStandsIds or sequenceStandIds?

**Review by lastminutediorama — APPROVED** [review]
lgtm - just asked a few questions in case they weren't intentional

File: src/interface/src/app/treatments/treatment-map/treatment-map.component.ts (hunk 3 above)
Comment (pflopez) [reply src/interface/src/app/treatments/treatment-map/treatment-map.component.ts:162]: placeholder, kinda hard to figure this out without the final patterns

File: src/interface/src/app/treatments/map-stands/map-stands.component.ts (hunk 4 above)
Comment (pflopez) [reply src/interface/src/app/treatments/map-stands/map-stands.component.ts:84]: hm which one do you think sounds better?

Comment (lastminutediorama) [reply src/interface/src/app/treatments/map-stands/map-stands.component.ts:84]: Either is okay, but I'd vote for sequenceStands for consistency

**Pull request merged**